packages/ui/src/components/sign-in/sign-in.tsx

```diff
@@ -521,8 +594,8 @@ export function SignInComponentLoaded() {
                     <Card.FooterActionText>
                       {t('signIn.alternativeMethods.actionText')}{' '}
                       <Card.FooterActionLink
-                        // To be implemented in SDKI-115
-                        href='#'
+                        as='button'
+                        onClick={() => setShowHelp(true)}
```

Comment:
Any strong reason against introducing a FooterActionButton instead of needing to make this a polymorphic component?